File: igrant_org_sdk/src/main/java/io/igrant/igrant_org_sdk/models/Organizations/OrganizationDetailResponse.java
package io.igrant.igrant_org_sdk.models.Organizations;

import io.igrant.igrant_org_sdk.models.Organization;

import java.util.ArrayList;

public class OrganizationDetailResponse {

    private Organization Organization;
    private String ConsentID;
    private ArrayList<PurposeConsent> PurposeConsents = null;

    public Organization getOrganization() {
        return Organization;
    }

    public void setOrganization(Organization Organization) {
        this.Organization = Organization;
    }

    public String getConsentID() {
        return ConsentID;
    }

    public void setConsentID(String ConsentID) {
        this.ConsentID = ConsentID;
    }

    public ArrayList<PurposeConsent> getPurposeConsents() {
        return PurposeConsents;
    }

    public void setPurposeConsents(ArrayList<PurposeConsent> PurposeConsents) {
        this.PurposeConsents = PurposeConsents;
    }

}

File: privacy-dashboard/src/main/java/com/github/privacydashboard/models/OrganizationDetailResponse.kt
package com.github.privacydashboard.models

import com.google.gson.annotations.SerializedName

data class OrganizationDetailResponse(
    @SerializedName("Organization")
    var organization: Organization? = null,
    @SerializedName("ConsentID")
    var consentID: String? = null,
    @SerializedName("PurposeConsents")
    var purposeConsents: ArrayList<PurposeConsent>? = null
) {}
